x-pack/plugins/watcher/public/lib/documentation_links/documentation_links.ts

```diff
@@ -12,6 +12,7 @@ const esStackBase = `${ELASTIC_WEBSITE_URL}guide/en/elastic-stack-overview/${DOC
 
 export const putWatchApiUrl = `${esBase}/watcher-api-put-watch.html`;
 export const executeWatchApiUrl = `${esBase}/watcher-api-execute-watch.html#watcher-api-execute-watch-action-mode`;
+export const watcherGettingStartedUrl = `${esStackBase}/watcher-getting-started.html`;
```

Comment:
I think it might be more helpful to link to the Watcher UI docs: https://www.elastic.co/guide/en/kibana/current/watcher-ui.html

In fact, we should update the screenshots in the docs as part of this work. Would you mind adding it as an item to the roadmap?

Reply:
done